### PR TITLE
Split test of quickstart examples into two separate jobs

### PR DESCRIPTION
**Pull request by maxalbert**
### I have:

- [ ] Formatted any Python files with [black](https://github.com/ambv/black)
- [X] Brought the branch up to date with master
- [X] Added any relevant Github labels
- [ ] Added tests for any new additions
- [ ] Added or updated any relevant documentation
- [ ] Added an Architectural Decision Record (ADR), if appropriate
- [ ] Added an [MPLv2 License Header](https://www.mozilla.org/en-US/MPL/headers/) if appropriate
- [ ] Updated the [Changelog](https://github.com/Flowminder/FlowKit/blob/master/CHANGELOG.md) 

### Description

This PR splits the test of the quickstart examples into two separate jobs, to reduce the build time by ~5 minutes (the time it takes to run the quickstart examples after the regular quickstart script has been run).

There may be a reason why we don't want this (maybe because it creates an additional job?) but I thought I'd throw it out there.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -816,6 +835,15 @@ workflows:
             - build_flowetl
             - build_examples
           <<: *run_always_org_context
+      - test_quickstart_examples_with_smaller_data:
+          requires:
+            - build_flowdb
```
Comment (greenape) [inline .circleci/config.yml]: Could use an anchor on the first job to save redeclaring the requires?

File: .circleci/config.yml [hunk 2]
```diff
@@ -588,6 +588,25 @@ jobs:
...
+          name: Check all containers are down
+          command: |
+            RUNNING=`docker ps -q`
+            if [[ "$RUNNING" != "" ]]
+            then
+              echo "Error: all containers should be down, but some are still running."
+              exit 1
+            fi
+
+  test_quickstart_examples_with_smaller_data:
```
Comment (greenape) [inline .circleci/config.yml]: Make it a parameterised job?

**Comment by maxalbert** [comment]
Thanks @greenape, very good suggestions to keep things tidier. I have changed the config accordingly. 👍

**Comment by maxalbert** [comment]
FYI, I have added a commented section for the quickstart tests with "larger data" because you had it in there before. Let me know if you'd prefer to remove it.

**Pull request merged**